Orange/widgets/data/owrank.py

```diff
@@ -610,6 +606,21 @@ def updateVisibleScoreColumns(self):
             self.ranksView.setColumnHidden(i + 1, not shown)
             self.ranksView.setColumnWidth(i + 1, 100)
 
+        if self.headerState[self.rankMode] is not None:
```

Comment:
This if condition seems to be unnecessary and if removed would make the code (together with the next if) a bit more readable.